goiscsi_types.go

```diff
@@ -18,20 +18,47 @@ type ISCSITransportName string
+	ISCSISessionState_FAILED ISCSISessionState = "FAILED"
+	// ISCSISessionState_FREE has been deprecated and will be removed in future release
+	ISCSISessionState_FREE ISCSISessionState = "FREE"
 
-	ISCSIConnectionState_FREE             ISCSIConnectionState = "FREE"
-	ISCSIConnectionState_TRANSPORT_WAIT   ISCSIConnectionState = "TRANSPORT WAIT"
-	ISCSIConnectionState_IN_LOGIN         ISCSIConnectionState = "IN LOGIN"
-	ISCSIConnectionState_LOGGED_IN        ISCSIConnectionState = "LOGGED IN"
-	ISCSIConnectionState_IN_LOGOUT        ISCSIConnectionState = "IN LOGOUT"
+	// ISCSIConnectionState_FREE has been deprecated and will be removed in future release
+	ISCSIConnectionState_FREE ISCSIConnectionState = "FREE"
+	// ISCSIConnectionState_TRANSPORT_WAIT has been deprecated and will be removed in future release
+	ISCSIConnectionState_TRANSPORT_WAIT ISCSIConnectionState = "TRANSPORT WAIT"
+	// ISCSIConnectionState_IN_LOGIN has been deprecated and will be removed in future release
+	ISCSIConnectionState_IN_LOGIN ISCSIConnectionState = "IN LOGIN"
+	// ISCSIConnectionState_LOGGED_IN has been deprecated and will be removed in future release
+	ISCSIConnectionState_LOGGED_IN ISCSIConnectionState = "LOGGED IN"
+	// ISCSIConnectionState_IN_LOGOUT has been deprecated and will be removed in future release
+	ISCSIConnectionState_IN_LOGOUT ISCSIConnectionState = "IN LOGOUT"
+	// ISCSIConnectionState_LOGOUT_REQUESTED has been deprecated and will be removed in future release
 	ISCSIConnectionState_LOGOUT_REQUESTED ISCSIConnectionState = "LOGOUT REQUESTED"
-	ISCSIConnectionState_CLEANUP_WAIT     ISCSIConnectionState = "CLEANUP WAIT"
+	// ISCSIConnectionState_CLEANUP_WAIT has been deprecated and will be removed in future release
+	ISCSIConnectionState_CLEANUP_WAIT ISCSIConnectionState = "CLEANUP WAIT"
 
-	ISCSITransportName_TCP  ISCSITransportName = "tcp"
+	// ISCSITransportName_TCP has been deprecated and will be removed in future release
+	ISCSITransportName_TCP ISCSITransportName = "tcp"
+	// ISCSITransportName_ISER has been deprecated and will be removed in future release
 	ISCSITransportName_ISER ISCSITransportName = "iser"
+
+	ISCSISessionStateLOGGEDIN ISCSISessionState = "LOGGED_IN"
+	ISCSISessionStateFAILED   ISCSISessionState = "FAILED"
+	ISCSISessionStateFREE     ISCSISessionState = "FREE"
+
+	ISCSIConnectionStateFREE            ISCSIConnectionState = "FREE"
+	ISCSIConnectionStateTRANSPORTWAIT   ISCSIConnectionState = "TRANSPORT WAIT"
+	ISCSIConnectionStateINLOGIN         ISCSIConnectionState = "IN LOGIN"
+	ISCSIConnectionStateLOGGEDIN        ISCSIConnectionState = "LOGGED IN"
+	ISCSIConnectionStateINLOGOUT        ISCSIConnectionState = "IN LOGOUT"
+	ISCSIConnectionStateLOGOUTREQUESTED ISCSIConnectionState = "LOGOUT REQUESTED"
+	ISCSIConnectionStateCLEANUPWAIT     ISCSIConnectionState = "CLEANUP WAIT"
+
+	ISCSITransportNameTCP  ISCSITransportName = "tcp"
+	ISCSITransportNameISER ISCSITransportName = "iser"
```

Comment:
I guess this is typo for user, can you correct it?

Reply:
This is transport type iser